src/Symfony/Component/DependencyInjection/Tests/Loader/YamlFileLoaderTest.php

```diff
@@ -45,26 +45,26 @@ public function testLoadFile()
             $this->fail('->load() throws an InvalidArgumentException if the loaded file does not exist');
         } catch (\Exception $e) {
             $this->assertInstanceOf('\InvalidArgumentException', $e, '->load() throws an InvalidArgumentException if the loaded file does not exist');
-            $this->assertEquals('The service file "foo.yml" is not valid.', $e->getMessage(), '->load() throws an InvalidArgumentException if the loaded file does not exist');
+            $this->assertEquals('The file "foo.yml" does not exist.', $e->getMessage(), '->load() throws an InvalidArgumentException if the loaded file does not exist');
         }
 
         try {
             $m->invoke($loader, 'parameters.ini');
-            $this->fail('->load() throws an InvalidArgumentException if the loaded file is not a valid YAML file');
+            $this->fail('->load() throws an InvalidArgumentException if the loaded file does not exist');
         } catch (\Exception $e) {
-            $this->assertInstanceOf('\InvalidArgumentException', $e, '->load() throws an InvalidArgumentException if the loaded file is not a valid YAML file');
-            $this->assertEquals('The service file "parameters.ini" is not valid.', $e->getMessage(), '->load() throws an InvalidArgumentException if the loaded file is not a valid YAML file');
+            $this->assertInstanceOf('\InvalidArgumentException', $e, '->load() throws an InvalidArgumentException if the loaded file does not exist');
+            $this->assertEquals('The file "parameters.ini" does not exist.', $e->getMessage(), '->load() throws an InvalidArgumentException if the loaded file does not exist');
         }
 
         $loader = new YamlFileLoader(new ContainerBuilder(), new FileLocator(self::$fixturesPath.'/yaml'));
 
-        foreach (array('nonvalid1', 'nonvalid2') as $fixture) {
```

Comment:
I guess that's because in line 38 the path in which the loader is looking for files ends with `ini` while it should be `yaml`.